src/DailyArtPromptApp.test.js

```diff
@@ -108,30 +81,18 @@ describe('mapStateToProps', () => {
     const result = mapStateToProps(state)
     expect(result.promptsImages).toEqual('promptsImages')
   })
-
 })
 
 describe('mapDispatchToProps', () => {
```

Comment:
I think you don't have mapDispatchToProps, right>

Reply:
this is true, i have deleted this test